pkg/cmd/config.go

```diff
@@ -136,6 +137,13 @@ func configPrintCmd() *cobra.Command {
 		defaultKubernetesVersion,
 		"Kubernetes version")
 
+	cmd.Flags().StringVarP(
+		&opts.ContainerLogMaxSize,
+		longFlagName(opts, "ContainerLogMaxSize"),
+		shortFlagName(opts, "ContainerLogMaxSize"),
+		"100Mi",
+		"ContainerLogMaxSize")
```

Comment:
Very tiny nit: I would put this at the end, so variables are grouped by priority/importance.

Reply:
fixed